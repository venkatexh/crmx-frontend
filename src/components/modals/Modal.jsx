import "../../sass/components/modals/modal.component.scss";

const Modal = () => {
  return (
    <div className={"modalContainer"}>
      <div className={"modal"}></div>
    </div>
  );
};

export default Modal;
