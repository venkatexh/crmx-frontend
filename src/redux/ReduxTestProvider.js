import { Provider } from "react-redux";

const ReduxTestProvider = ({ children, reduxStore }) => (
  <Provider store={reduxStore}>{children}</Provider>
);

export default ReduxTestProvider;
