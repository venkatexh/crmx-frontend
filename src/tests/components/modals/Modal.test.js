import { shallow } from "enzyme";
import React from "react";
import Modal from "../../../components/modals/Modal";
import ReduxTestProvider from "../../../redux/ReduxTestProvider";

it("should be modal component snapshot", () => {
  expect(
    shallow(
      <ReduxTestProvider>
        <Modal />
      </ReduxTestProvider>
    )
  ).toMatchSnapshot();
});
