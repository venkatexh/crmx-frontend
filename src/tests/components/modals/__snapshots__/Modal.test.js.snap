// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be modal component snapshot 1`] = `ShallowWrapper {}`;
